src/common/boot_uf2_headers/include/boot/uf2.h

```diff
@@ -26,6 +26,8 @@
 #define UF2_FLAG_MD5_PRESENT             0x00004000u
 #define UF2_FLAG_EXTENSION_FLAGS_PRESENT 0x00008000u
 
+#define CYW43_FIRMWARE_FAMILY_ID    0xe48bff55u
```

Comment:
Just out of curiosity, why has this been added "below" `RP2040_FAMILY_ID` rather than "above" `RP2350_ARM_NS_FAMILY_ID` ?

And tangentially, I wonder if it'd be better to do
```c
#define RP2040_FAMILY_ID 0xe48bff56u
#define DEFAULT_FAMILY_ID_MIN RP2040_FAMILY_ID

#define RP2350_ARM_NS_FAMILY_ID 0xe48bff5bu
#define FAMILY_ID_MAX RP2350_ARM_NS_FAMILY_ID
```
in order to eliminate the duplicated hard-coded values? :thinking: